src/objs/basic_types.md

```diff
@@ -0,0 +1,76 @@
+## Basic Object Types
+
+In Object Oriented programming, objects inherit behaviour from parents. DM has some built in ancestors which share certain behaviours. These will be gone over individually and in more detail in their own pages, but here is the general overview.
+
+### Datums
+
+The first is the Datum (`/datum/`). It's the ancestor of all the other types (except for some types like /world, /client, /list, and more) that we will see. It's essentially pure data, hence the name, and can have pretty much any vars you like.
+> Note: When you define a new "top level" object, if you do not specify a parent_type, it defaults to /datum.
+
+```dm
+MyType
+	var/myvar = "test"
+// this "mytype object" is a datum. Normally, though, you'd put it as datum/MyType or /datum/MyType
+```
+
+### Atoms
+
+Atoms (`/atom/`) are the direct child of datums, and they represent objects that can go on a map and therefore in the world. It's an abstract parent which shouldn't really be used by itself, but it gives important information to its children, mainly **a**reas, **t**urfs, **o**bjs, and **m**obs.
+`/atom/movable` defines behaviour related to moving things, and is the parent of `/obj` and `/mob`. Again, this is an abstract parent, so you shouldn't define objects under it.
```

Comment:
Why not? Goon does.

Reply:
since when 💀 
i'll remove the abstract parent warning then